test/unittest/rtps/history/ReaderHistoryTests.cpp

```diff
@@ -55,7 +55,7 @@ class ReaderHistoryTests : public Test
 
     virtual void SetUp()
     {
-        history_attr.memoryPolicy = MemoryManagementPolicy_t::PREALLOCATED_MEMORY_MODE;
+        history_attr.memoryPolicy = MemoryManagementPolicy_t::PREALLOCATED_WITH_REALLOC_MEMORY_MODE;
```

Comment:
As the Memory Management Policy is explicitly set, the default behavior is not being tested. I suggest keeping the test as it was. Maybe we should consider parameterizing the test and check every Memory Management Policy.